src/data/set/basic.lean

```diff
@@ -909,7 +909,7 @@ ext $ λ x, and_iff_not_or_not
 
 @[simp] theorem compl_union_self (s : set α) : sᶜ ∪ s = univ := by rw [union_comm, union_compl_self]
 
-theorem compl_comp_compl : compl ∘ compl = @id (set α) := funext compl_compl
+theorem compl_comp_compl [boolean_algebra α] : compl ∘ compl = @id α := funext compl_compl
```

Comment:
This should go in the same file as `compl_compl`, and not in this file about sets

Reply:
In fact, currently this PR leaves the name as `set.compl_compl`!